corehq/apps/locations/templates/locations/manage/locations.html

```diff
@@ -114,9 +114,11 @@
 <div class="row-fluid">
     <div class="span12">
         <div class="btn-toolbar">
-            <a class="btn btn-info" href="{% url "location_import" domain %}">
-                <i class="icon-upload"></i> {% trans 'Bulk Import Locations' %}
-            </a>
+            {% if has_location_types %}
+                <a class="btn btn-info" href="{% url "location_import" domain %}">
+                    <i class="icon-upload"></i> {% trans 'Bulk Import Locations' %}
```

Comment:
related to other comment - might be nice to disable the button with hover text explaining why